public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>Pictionary</title>
+    <link rel="stylesheet" href="build/pictionary.1.0.0.min.css">
```

Comment:
changed to `pictionary.1.0.0.min.css`. Already serving from `build` directory no need to keep it in the path.